contracts/proof/RLP.sol

```diff
@@ -187,9 +189,10 @@ library RLP {
  /// @return Array of RLPItems.
  function toList(RLPItem memory self) internal pure returns (RLPItem[] memory list) {
      require(isList(self));
-     var numItems = items(self);
+     uint numItems;
```

Comment:
Why not declare and define at the same time?

Reply:
you are right.
Made the change.